config/config.go

```diff
@@ -787,6 +791,21 @@ RootDir: "{{ .RootDir }}"
 DBName: "{{ .DBName }}"
 GenesisFile: "{{ .GenesisFile }}"
 PluginsDir: "{{ .PluginsDir }}"
+
+{{if .DPOS -}}
+#
+# Configuration of DPOSv3
+#
+DPOS:
+  # Specifies addresses of bootstrap nodes
+  BootstrapNodes:
+  {{- range .DPOS.BootstrapNodes}}
+    - "{{. -}}"
+  {{- end}}
+  # Time duration in seconds in which TotolStaked is valid  
```

Comment:
-> `How long (in seconds?) the response from the dpos_total_staked RPC method should be cached.`

Reply:
Done